tests/scripts/test_scripts.py

```diff
@@ -882,13 +883,13 @@ def test_parallel_train_adversarial_custom_env(tmpdir):
 
     config_updates = dict(
         sacred_ex_name="train_adversarial",
-        n_seeds=1,
+        repeat=2,
         base_named_configs=[env_named_config] + ALGO_FAST_CONFIGS["adversarial"],
         base_config_updates=dict(
             logging=dict(log_root=tmpdir),
             demonstrations=dict(path=path),
         ),
-        search_space=dict(command_name="gail"),
+        search_space=dict(command_name=tune.choice(["gail"])),
```

Comment:
Why was this needed? Can we document this inline?

Reply:
This was needed because the search space needs to contain at least one element (so that it can be sampled) when we use the optuna search algorithm (enabled by repeat=2).

Reply:
All right. Then please document this inline because somebody (probably me) will be confused by this construct when I read it again :-)

Reply:
Done 👍